test/application_system_test_case.rb

```diff
@@ -3,10 +3,10 @@
 require 'test_helper'
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
-  driven_by :selenium, using: :headless_chrome, screen_size: [1920, 1080], options: {
+  driven_by :selenium, using: :chrome, screen_size: [1920, 1080], options: {
     desired_capabilities: {
       chromeOptions: {
-        args: %w[headless disable-gpu disable-dev-shm-usage]
+        args: %w[no-sandbox headless disable-gpu disable-dev-shm-usage]
```

Comment:
`disable-dev-shm-usage` bunu kaldırarak denedik mi?  hatırladığım kadarıyla memory sorunları yaşadığımızdan circle tarafında eklemiştik bunu, travis'de gerekli olmayabilir.  shm geçici dosyalar için bellekteki temp aygıt oluyor.  yeterli bellek varsa dursun bence, işleri bir parça hızlandırabilir.

Reply:
> By default, Docker runs a container with a /dev/shm shared memory space 64MB. This is typically too small for Chrome and will cause Chrome to crash when rendering large pages. 
https://developers.google.com/web/tools/puppeteer/troubleshooting

Bu ayarla şimdi çalışsa bile ilerde patlama riski varmış benim anladığım.